README.md

```diff
@@ -164,6 +164,28 @@ python util/taskcluster.py --target /tmp --source tensorflow --arch gpu --artifa
 pip install /tmp/tensorflow_gpu_warpctc-1.3.0rc0-cp27-cp27mu-linux_x86_64.whl
 ```
 
+### Common Voice training data
+
+The Common Voice corpus consists of voice samples that were donated by the [Common Voice web application](https://voice.mozilla.org/).
+For automatically downloading and importing the corpus into a given directory, you can call:
+
+```bash
+bin/import_cv.py path/to/a/directory
+```
+
+Please be aware that this requires at least 70GB of free disk space and quite some time to conclude. 
+As this process creates a huge number of small files, using a SSD drive is highly recommended.
+To some extend the importer is reentrant and tries its best to continue where it got interrupted. 
+Once the import is done, the directory will contain a bunch of CSV files:
+`cv-valid-train.csv`, `cv-valid-dev.csv` and `cv-valid-test.csv` are the official user-validated sets for training, validation and test respectively.
+`cv-other-train.csv`, `cv-other-dev.csv` and `cv-other-test.csv` are the non-validated unofficial sets for training, validation and test respectively.
+`cv-invalid.csv` is containing all samples that users qualified as invalid.
+
+A sub-directory prefixed with `cv_corpus_` contains the extracted mp3 and wav files that were extracted from an archive with the same prefix.
```

Comment:
It would be good to call out that version number is embedded here. I think we can stick to this convention going forward:
"A sub-directory called `cv_corpus_v{version}` where version equals the version number of the Common Voice corpus you downloaded, contains....